bindings/python/tests/CMakeLists.txt

```diff
@@ -34,9 +34,9 @@ function(test_python_bindings test)
 	CONFIGURATION ${conf})
       set_tests_properties(${test}_${conf}_py
     	PROPERTIES ENVIRONMENT
-        PYTHONPATH="${PROJECT_BINARY_DIR}/bindings/python/tests/;$ENV{PYTHONPATH}"
-        MGIS_TEST_BEHAVIOURS_LIBRARY="$<TARGET_FILE:BehaviourTest>"
-        MGIS_TEST_MODELS_LIBRARY="$<TARGET_FILE:ModelTest>")
+        "PYTHONPATH=${PROJECT_BINARY_DIR}/bindings/python/tests/;$ENV{PYTHONPATH}"
```

Comment:
@bilke Could you comment on why you did this change. Is that what is broken now ?

Reply:
Without this change I get this error:

```
CMake Error at /cache/cpm/mgis/d91b741daaf78d4ef414125d6c96d30479c88d53/bindings/python/tests/CMakeLists.txt:35 (set_tests_properties):
  set_tests_properties called with incorrect number of arguments.
Call Stack (most recent call first):
  /cache/cpm/mgis/d91b741daaf78d4ef414125d6c96d30479c88d53/bindings/python/tests/CMakeLists.txt:53 (test_python_bindings)
```

I think as the value of `ENVIRONMENT` should be a list the problem is that `PYTHONPATH="${PROJECT_BINARY_DIR}/bindings/python/tests/;$ENV{PYTHONPATH}"` is not identified as a list item.

Reply:
@bilke. Could you give information about your system ?

Reply:
@thelfer I think I got the issue corrected now: there was a semicolon used as a path separator, which confused CMake (as `;` is also the list separator in CMake) and should be `:` instead.

Reply:
@bilke Nice. I believe that the `;` separator comes from `Windows`.  I need to check that. But for the moment, I merge the PR. Thanks again !